packages/@react-aria/interactions/src/usePress.ts

```diff
@@ -466,7 +466,7 @@ export function usePress(props: PressHookProps): PressResult {
           }
         } else if (state.isOverTarget) {
           state.isOverTarget = false;
-          triggerPressEnd(createEvent(state.target, e), state.pointerType, false);
+          triggerPressEnd(createEvent(state.target, e), state.pointerType);
```

Comment:
Won't this mean we trigger onPress when the user just presses down and then drags their mouse off without releasing? I think that's what the tests are catching as well.

Reply:
I don't think so, here's a video, I pressed down the NextUI Button (custom usePress) and the `onPress` is not triggered when dragging without release, it seems to have the same behaviour.
 
https://github.com/adobe/react-spectrum/assets/30373425/a74baf7a-6192-4b5a-bfb6-56eeab9abd83

Here's the version with custom `usePress`:
https://nextui-storybook-v2-git-feat-autocomplete-nextui-org.vercel.app/?path=%2Fstory%2Fcomponents-button--with-state&vercelToolbarCode=e1L8-1_7if1HslV

Here's the version with react-aria `usePress`:
https://storybook.nextui.org/?path=/story/components-button--with-state

I don't see any difference

Reply:
I'm not sure what's going on in nextui, but I just tried out this branch locally with our Button story in storybook and `press` action is fired if I mouse down and drag off the button.

Reply:
Here's the code I implemented on NextUI https://github.com/nextui-org/nextui/pull/1797/files#diff-50bba2c230beaca7e521bcf1b5c27aba093884f9d15ee05011468abd71c7eda2

Reply:
could you do a diff between your version of usePress and the one in this PR to see what the difference is?

Reply:
Hey, I did a diff of the file you linked to and our code. I do not see the change you're proposing in this PR in that diff...
From what I can tell, you meant to put it one more function down, in `onPointerUp`?